hca/util/__init__.py

```diff
@@ -225,6 +225,12 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 class _PaginatingClientMethodFactory(_ClientMethodFactory):
     def iterate(self, **kwargs):
+        """
+        Yield specific items from each response depending on its contents.
+
+        For example, bundles and collections yield the files contained within and
+        a POST search yields result items.
```

Comment:
Consider:

"For example, `GET /bundles/{id}` and `GET /collections/{id}` yield the items contained within; `POST /search` yields search result items."